### PR TITLE
feat: Makefile target matching GH linters

### PR DESCRIPTION
**Pull request by Ropes**
Provides the ability to run the same golangci-lint tests which are
triggered for every PR.

~~`make test-gh-lint`~~ `make test-sublinters` utilizes the same config file as GH tests. Two of
the linters are disabled same as they are for GH PRs with
`continue-on-error`.

When a linter check is broken, it fails the Make target and prints cause
of failure.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -79,10 +79,54 @@ test-coverage:
...
+	#golangci-lint run ./... --disable-all --enable gocritic
+	golangci-lint run ./... --disable-all --enable goconst
+	#golangci-lint run ./... --disable-all --enable gochecknoglobals
+	golangci-lint run ./... --disable-all --enable ineffassign --no-config
+	golangci-lint run ./... --disable-all --enable unparam
+	golangci-lint run ./... --disable-all --enable staticcheck
+	golangci-lint run ./... --disable-all --enable golint
+	golangci-lint run ./... --disable-all --enable gosec
+	golangci-lint run ./... --disable-all --enable scopelint
+	golangci-lint run ./... --disable-all --enable prealloc
```
Comment (boz) [inline Makefile:98]: remove :point_up: and use make targets in the gh workflow?

**Comment by Ropes** [comment]
Will clean up and rebase once https://github.com/ovrclk/akash/pull/573 is merged.

**Comment by Ropes** [comment]
ping @boz, I think we're good to go!

**Comment by boz** [comment]
@Ropes still waiting for it to be used in the GH workflow so that the two don't get out of sync.

**Comment by Ropes** [comment]
@boz GH action now uses `make test-sublinters` target in tests.yml.

**Pull request merged**